portfolio/src/main/webapp/script.js

```diff
@@ -35,18 +35,22 @@ async function showServerTime() {
 }
 
 /** Adds random quotes to the html page. */
-async function printString() {
+async function printRandomQuote() {
     const getResponseFromServer = await fetch('/quotes');
-    const jsonFromResponse = await getResponseFromServer.json();
-    console.log(jsonFromResponse);
+    const quotes = await getResponseFromServer.json();
 
-    const stringContainer = document.getElementById('life-quote-container');
-    stringContainer.innerHTML = getRandomQuote(jsonFromResponse);
+    const quote = quotes[Math.floor(Math.random() * quotes.length)];
+
+    const quoteContainer = document.getElementById('life-quote-container');
+    quoteContainer.innerHTML = quote;
 }
 
-/** Gets random quote from server. */
-function getRandomQuote(jsonFromResponse) {
-    const randomIndex = Math.floor(Math.random() * jsonFromResponse.quotes.length);
-    const randomQuote = jsonFromResponse.quotes[randomIndex];
```

Comment:
jsonFromResponse.quotes was the bug I think. It should be just jsonFromResponse which you do here in this commit ("quotes[Math.floor(Math.random() * quotes.length)];").